README.md

```diff
@@ -3,8 +3,8 @@
 
 This package supports running Cypress in GoCD by offering the following features:
 
-- start the local development server of your application
 - run testing in a Docker container when the `CI` environment flag is set
+- optionally start the local development server of your application
 - creates a merged HTML report
```

Comment:
Since now we're listing features instead of telling what GoCD does, we should say "create" instead of "creates" to be consistently correct here.